dcicutils/glacier_utils.py

```diff
@@ -342,12 +363,18 @@ def copy_object_back_to_original_location(self, bucket: str, key: str, storage_c
             response = self.s3.head_object(Bucket=bucket, Key=key)
             size = response['ContentLength']
             multipart = (size >= MAX_STANDARD_COPY_SIZE)
+            if not preserve_lifecycle_tag:  # default: preserve tags except 'Lifecycle'
+                tags = self.s3.get_object_tagging(Bucket=bucket, Key=key).get('TagSet', [])
+                tags = [tag for tag in tags if tag['Key'] != ENCODED_LIFECYCLE_TAG_KEY]
+                tags = self._format_tags(tags)
```

Comment:
For files for which the only existing tag is the lifecycle tag, this results in `tags = ""`, which unfortunately then does not clear the tag. If the truthiness is `False`, something like this should work instead:
```suggestion
                tags = self.s3.get_object_tagging(Bucket=bucket, Key=key).get('TagSet', [])
                tags = [tag for tag in tags if tag['Key'] != ENCODED_LIFECYCLE_TAG_KEY]
                tags = self._format_tags(tags)
                if not tags:
                    self.s3.delete_object_tagging(Bucket=bucket, Key=key)
```
I used the above for some existing files, and the tags were cleared appropriately.